test/mocha/10-keystore.js

```diff
@@ -3,8 +3,12 @@
  */
 'use strict';
 
+const bedrock = require('bedrock');
 const {CapabilityAgent} = require('webkms-client');
 const helpers = require('./helpers');
+const brHttpsAgent = require('bedrock-https-agent');
+const axios = require('axios');
```

Comment:
all new code should use @digitalbazaar/http-client,  Here's some code that uses it for examples: https://github.com/digitalbazaar/bedrock-rsvp/blob/271ad0b64973bc51e3bbf70e8f7d722f6343f512/test/mocha/50-http.js#L1

Reply:
Updated to use httpClient instead of axios.